ui/src/flux/components/DatabaseListItem.tsx

```diff
@@ -83,6 +81,44 @@ class DatabaseListItem extends PureComponent<Props, State> {
     })
   }
 
+  private get filterAndTagList(): JSX.Element {
+    const {db, service} = this.props
+    const {isOpen, searchTerm} = this.state
+
+    if (isOpen) {
+      return (
+        <>
+          <div className="flux-schema--filter">
+            <input
+              className="form-control input-sm"
+              placeholder={`Filter within ${db}`}
+              type="text"
+              spellCheck={false}
+              autoComplete="off"
+              value={searchTerm}
+              onClick={this.handleInputClick}
+              onChange={this.onSearch}
+            />
+          </div>
+          <TagList db={db} service={service} tags={this.tags} filter={[]} />
+        </>
+      )
+    }
+  }
+
+  private handleCopyClick = e => {
+    e.stopPropagation()
+  }
+
+  private handleCopy = (copiedText: string, isSuccessful: boolean): void => {
```

Comment:
1. this method doesn't appear to do anything with the `copiedText` besides generate a notification, and it appears from the package docs that this is the case. so i would contend that this is not really a handler for the copy action itself, but rather a secondary handler that is triggered when the copy is attempted. so what do you think about naming it something like `handleCopyAttempted` for clarity? as-is, i wondered why only notifications were being generated from what otherwise seemed like an action handler. in other words, even if there were no handler, the copy action would still have happened (if successful). so what do you think about naming the function in a way that captures this nuance?
1. additionally, what do you think about naming this method in such a way as to bake in more specificity about what it's doing and why? i ask because it's a method on a component that doesn't only have to do with copying, so a method on such a class called `handleCopy` doesn't register within my expectations of what i'd find on such a component class, and so i don't know what copying is in this context. something like `handleCopyTextToClipboardAttempted`. i'm not clear, at least from just reading the code, what is being copied, and to where, so my personal feeling is that the name could communicate the context to me so that i don't have to try to figure it out

Reply:
It's been renamed to `handleCopyAttempt` for clarity